client/my-sites/sites/sites.jsx

```diff
@@ -47,7 +47,7 @@ export const Sites = React.createClass( {
 
 		// Filter out sites with no upgrades on particular routes
 		if ( /^\/domains/.test( path ) || /^\/plans/.test( this.props.sourcePath ) ) {
-			return this.props.isSiteUpgradeable( site.ID );
+			return site.isSiteUpgradeable !== false;
```

Comment:
After further consideration, I've decided to go with your suggestion here, it makes it much more readable than what I suggested.